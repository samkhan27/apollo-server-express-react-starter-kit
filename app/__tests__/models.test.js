const { getAllBooks } = require('../graphql/models')

test('it returns correct books', () => {
    expect(getAllBooks()).toMatchSnapshot()
})