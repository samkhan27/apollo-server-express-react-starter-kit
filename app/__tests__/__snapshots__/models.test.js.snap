// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it returns correct books 1`] = `
Array [
  Object {
    "author": "J.K. Rowling",
    "title": "Harry Potter and the Chamber of Secrets",
  },
  Object {
    "author": "Michael Crichton",
    "title": "Jurassic Park",
  },
]
`;
